packages/ui/components/images/blockchain/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { View, Image, Text } from 'react-native'
+
+import { calculateImageSize } from 'ui/components/images/lib/image-size'
+
+import style from 'theme/image'
+
+interface Props {
+  size: number
+}
+
+export default ({size} : Props) => (
+  <View>
+    <Image
+      style={[style.image, calculateImageSize(size)]}
+      source={require('images/blockchain.jpg')}
```

Comment:
Instead of having all these different components, have one `ThemeImage` component in a file called `ui/components/theme-image` that takes a `name` prop. Then: ```require(`images/${name}.jpg`)``` - bam just deduplicated a ton of duplicated code.